sql/updates/latest-dev.sql

```diff
@@ -151,12 +151,31 @@ FROM _timescaledb_config.bgw_policy_reorder reorder
 WHERE job_type = 'reorder'
   AND job.id = reorder.job_id;
 
+-- migrate compression jobs
+UPDATE
+  _timescaledb_config.bgw_job job
+SET proc_name = 'policy_compression',
+  proc_schema = '_timescaledb_internal',
+  config = jsonb_build_object('hypertable_id', c.hypertable_id, 'older_than', CASE WHEN (older_than).is_time_interval THEN (older_than).time_interval::text ELSE (older_than).integer_interval::text END),
+  hypertable_id = c.hypertable_id,
+  OWNER = (
```

Comment:
Why is owner in allcaps?

Reply:
that was pg_formats doing because its SQL keyword